File: code/basics/range-operator.cpp
#include &lt;iostream&gt;

// C++ doesn't have a range operator like some languages
// Use loops or std::iota with containers instead
int main() {
    for (int i = 0; i &lt; 5; i++) {
        std::cout &lt;&lt; i &lt;&lt; std::endl;
    }
    return 0;
}

File: code/basics/range-operator.erl
% Range operator in Erlang
% Erlang doesn't have a built-in range operator
% Use lists or recursion
-module(example).
-export([range/2, start/0]).

range(Start, End) when Start &gt; End -&gt;
    [];
range(Start, End) -&gt;
    [Start | range(Start + 1, End)].

start() -&gt;
    Range = range(0, 4),
    [io:format("~p~n", [X]) || X &lt;- Range].